### ISSUE TITLE
VMware ESXi 7.0 Update 1 SLIC 2.5 / macOS Unlock MOD ISO - sysin - System Inside

### ISSUE THREAD

**Issue by netgc** [netgc/gitalk#39]
https://sysin.org/article/vmware-esxi-7-u1-slic/ 

  VMware ESXi 7.0 Update 1 SLIC 2.5 / macOS Unlock MOD ISO - sysin | System Inside | 数据中心：系统、平台和应用交付

**Comment by wjx0912** [comment]
能不能放个unlocker 出来。已经安装esxi7.01u了。（手上没有mac没法制作esxi-unlocker）谢谢啦

**Comment by wjx0912** [comment]
@netgc  谢谢回复。

我的意思是能不能放出esxi-unlocker.tgz文件。因为你发的是iso，需要重新安装esxi了，我的esxi7.01u运行有一段时间了，里面的数据不方便重装。

可以再另外写一篇已经装好esxi701u的情况下解锁mac的文章吗。万分感谢~~

**Comment by netgc** [comment]
> @wjx0912
> 能不能放个unlocker 出来。已经安装esxi7.01u了。（手上没有mac没法制作esxi-unlocker）谢谢啦

下载这个脚本：[esxi-unlocker-3.0](https://github.com/netgc/esxi-unlocker-3.0)，上传到ESXi比如/tmp目录，执行./esxi-install.sh，重启即可。

**Comment by wjx0912** [comment]
@netgc   

感谢答复。 
我的意思是能不能用esxi-install.sh生成一个esxi-unlocker-302.tgz，  现在github搜索的都是老版本的esxi-unlocker-301.tgz（支持esxi7.0，但不支持esxi7.0u1）

这个文件貌似要在macos下生成。我拿到linux下强行改了esxi-install.sh报错的地方，然后esxi里面执行esxi-smctest.sh也报错了。。。

万分感谢~~

**Comment by netgc** [comment]
> @wjx0912
> @netgc   
> 
> 感谢答复。 
> 我的意思是能不能用esxi-install.sh生成一个esxi-unlocker-302.tgz，  现在github搜索的都是老版本的esxi-unlocker-301.tgz（支持esxi7.0，但不支持esxi7.0u1）
> 
> 这个文件貌似要在macos下生成。我拿到linux下强行改了esxi-install.sh报错的地方，然后esxi里面执行esxi-smctest.sh也报错了。。。
> 
> 万分感谢~~

[Release v3.0.2](https://github.com/netgc/esxi-unlocker-3.0/releases/tag/3.0.2)

**Comment by wjx0912** [comment]
谢谢

**Comment by wx77** [comment]
你好，标准镜像 VMware-VMvisor-Installer-7.0U1-16850804.x86_64-SLIC-sysin.org.iso 为什么没办法直接启动安装呢？

**Comment by netgc** [comment]
> @wx77
> 你好，标准镜像 VMware-VMvisor-Installer-7.0U1-16850804.x86_64-SLIC-sysin.org.iso 为什么没办法直接启动安装呢？

你好，经过测试，确实发现在 EFI 引导时候无法启动，使用 BIOS 引导正常，将重新发布（预计今天晚些时候），修正 EFI 引导问题，谢谢。

**Comment by netgc** [comment]
> @wx77
> 你好，标准镜像 VMware-VMvisor-Installer-7.0U1-16850804.x86_64-SLIC-sysin.org.iso 为什么没办法直接启动安装呢？

已更新下载链接，谢谢。

**Comment by imfiddle** [comment]
我是小白一个，在ThinkServerRD640上安装7.01时，使用联想定制版还是找不到磁盘。

**Comment by netgc** [comment]
> @imfiddle
> 我是小白一个，在ThinkServerRD640上安装7.01时，使用联想定制版还是找不到磁盘。

经查询，该机型官方仅支持到 5.5 版本，详见：[https://lenovopress.com/osig](https://lenovopress.com/osig)

**Comment by osmanage** [comment]
请问这个只能用于7u1不能用于6.7u3是吧？

**Comment by osmanage** [comment]
由于7u1不支持我的双口intel PT网卡，所以我用的6.7u3，我下载并安装了3.02脚本，重启后没有在创建虚拟机列表中发现mac的10.16版本。我应该怎么办？和之前一样只有10.14，没有改变什么，我不大明白那个脚本的作用是什么？

**Comment by osmanage** [comment]
那既然我安装unlocker脚本之前 esxi6.7u3就存在macos 10.14的选项，是不是意味着不安装unlocker也自带了macos 10.14的虚拟机支持呢？安装unlocker多余了？反正也支持不了10.16 big sur？

**Comment by osmanage** [comment]
我运行 esxi-smctest后，仍旧仅显示 smcPresent = false ，插件重启后没有起作用。（对6.7u3无效？）还是6.7u3已默认支持到10.14不需要且unlocker无意义了？

**Comment by osmanage** [comment]
HP的服务器ESXi镜像里面是多了什么“好东西”，比其它镜像大了那么多呢？

**Comment by netgc** [comment]
> @osmanage
> HP的服务器ESXi镜像里面是多了什么“好东西”，比其它镜像大了那么多呢？

详见：[官网文档](https://support.hpe.com/hpesc/public/docDisplay?docId=c04430318)，7.0 U1 Oct 2020 部分描述内容。

**Comment by netgc** [comment]
> @osmanage
> 我运行 esxi-smctest后，仍旧仅显示 smcPresent = false ，插件重启后没有起作用。（对6.7u3无效？）还是6.7u3已默认支持到10.14不需要且unlocker无意义了？

这里仅仅针对和讨论的7.0U1，其他版本不适用或者无关（包括描述的几个特性）。

**Comment by osmanage** [comment]
> @netgc
> > @osmanage
> > 我运行 esxi-smctest后，仍旧仅显示 smcPresent = false ，插件重启后没有起作用。（对6.7u3无效？）还是6.7u3已默认支持到10.14不需要且unlocker无意义了？
> 
> 这里仅仅针对和讨论的7.0U1，其他版本不适用或者无关（包括描述的几个特性）。

！！您真是门儿清每个细节！  HP集成了好多驱动。  请问我主板是SuperMicro的，是否只能安装标准版，定制版确实是要对应品牌的服务器硬件才适合安装？

**Comment by netgc** [comment]
> @osmanage
> > @netgc
> > > @osmanage
> > > 我运行 esxi-smctest后，仍旧仅显示 smcPresent = false ，插件重启后没有起作用。（对6.7u3无效？）还是6.7u3已默认支持到10.14不需要且unlocker无意义了？
> > 
> > 这里仅仅针对和讨论的7.0U1，其他版本不适用或者无关（包括描述的几个特性）。
> 
> ！！您真是门儿清每个细节！  HP集成了好多驱动。  请问我主板是SuperMicro的，是否只能安装标准版，定制版确实是要对应品牌的服务器硬件才适合安装？

已知 SuperMicro 并没有在 VMware 发布 7.0 版本的定制镜像，可以尝试标准版，具体机型是否支持和支持的版本需要搜索 [VMware HCL](https://www.vmware.com/resources/compatibility/search.php)，当然上面即使没有列出也可以尝试，这里只表示官方的支持。

**Comment by ecar1128** [comment]
安装完dell的esxi7.0 u1之后。v'm'tools不能正常安装。提示找不到安装文件。iso没有加载。什么原因？

**Comment by osmanage** [comment]
我安装了标准版，请问Windows 2019的SLIC 2.5支持自动激活的话，是否需要特殊版本的Windows2019ISO安装才可以利用BIOS SLIC激活？（因为我安装了msdnitellyou的原版没有自动激活）        当然，我也下载了你定制的Win2019 ova文件进行安装，但是肯定存在问题，这个应该不是我才遇到的问题，该Win2019 ova尚有不妥，我在Win2019对应的那篇文章中向您提出。

**Comment by netgc** [comment]
> @ecar1128
> 安装完dell的esxi7.0 u1之后。v'm'tools不能正常安装。提示找不到安装文件。iso没有加载。什么原因？
> 

经测，使用一台 Windows Server 2019 虚机，未安装 VM-Tools，点击安装可以正常挂载 ISO，自动弹出安装界面。已经安装旧版的 VM-Tools，点击升级，可以自动完成升级。没有出现上述问题。

**Comment by netgc** [comment]
> @osmanage
> 我安装了标准版，请问Windows 2019的SLIC 2.5支持自动激活的话，是否需要特殊版本的Windows2019ISO安装才可以利用BIOS SLIC激活？（因为我安装了msdnitellyou的原版没有自动激活）        当然，我也下载了你定制的Win2019 ova文件进行安装，但是肯定存在问题，这个应该不是我才遇到的问题，该Win2019 ova尚有不妥，我在Win2019对应的那篇文章中向您提出。


> @osmanage
> 我安装了标准版，请问Windows 2019的SLIC 2.5支持自动激活的话，是否需要特殊版本的Windows2019ISO安装才可以利用BIOS SLIC激活？（因为我安装了msdnitellyou的原版没有自动激活）        当然，我也下载了你定制的Win2019 ova文件进行安装，但是肯定存在问题，这个应该不是我才遇到的问题，该Win2019 ova尚有不妥，我在Win2019对应的那篇文章中向您提出。

对 Windows 2019 的版本没有要求，需要满足三个条件：BIOS 中的 SLIC（ESXi 这里的镜像解决这个问题）、厂商的 OEM 证书和 KEY（序列号），后面两项可以搜索一下。
